wcivf/apps/parties/templates/parties/party_detail.html

```diff
@@ -22,63 +22,82 @@
+                    </div>
+                    <div class="ds-not-sidebar">
                         <h2>{{ party.party_name }}</h2>
-                        <p>{% blocktrans trimmed with num_candidates=object.personpost_set.all.count|intcomma pluralize_candidates=object.personpost_set.all.count|pluralize %}{{ num_candidates }} candidate{{ pluralize_candidates }}{% endblocktrans %}</p>
+                        <p>{% blocktrans with num_candidates=object.personpost_set.all.count|intcomma pluralize_candidates=object.personpost_set.all.count|pluralize %}{{ num_candidates }} candidate{{ pluralize_candidates }} in our database.{% endblocktrans %}</p>
                         {% if object.description %}
                             {{ object.description|markdown }}
                         {% endif %}
                         {% if object.wikipedia_url %}
                             <p><a href="{{ object.wikipedia_url }}">{% trans "Read more on wikipedia" %}</a></p>
                         {% endif %}
                     </div>
-
+                </div>
+            </div>
+        {% else %}
+            <div>
+                <h2>{{ object.format_name }}</h2>
+                <p>{% blocktrans trimmed with num_candidates=object.personpost_set.all.count|intcomma pluralize_candidates=object.personpost_set.all.count|pluralize %}{{ num_candidates }} candidate{{ pluralize_candidates }} in our database.{% endblocktrans %}</p>
+                {% if object.description %}
+                    {{ object.description|markdown }}
+                {% endif %}
+                {% if object.wikipedia_url %}
+                    <p><a href="{{ object.wikipedia_url }}">{% trans "Read more on wikipedia" %}</a></p>
+                {% endif %}
+                {% if object.is_deregistered %}
+                    <p>{% blocktrans with dereg_date=object.date_deregistered %}This political party was deregistered on {{ dereg_date }}, and can no longer field candidates. {% endblocktrans %}</p>
                 {% endif %}
+            </div>
+        {% endif %}
+        {% if object.is_joint_party %}
+            {% with sub_parties=object.get_joint_party_sub_parties %}
+                <div class="ds-card">
+                    <div class="ds-card-body">
+                        <h3>
+                            <span aria-hidden="true">🤝</span>
+                            {% trans "Joint description" %}
+                        </h3>
+                        <p>
+                            {% blocktrans with joint_name=object.party_name first_name=sub_parties.0.party_name second_name=sub_parties.1.party_name %}
+                                "{{ joint_name }}" is a joint description of the {{ first_name }} and the {{ second_name }}.
+                            {% endblocktrans %}
+                        </p>
+                    </div>
+                </div>
+                <div>
```

Comment:
This and below: I'm not sure this empty div is doing anything?

Reply:
You're right. I put these in divs earlier to get the `ds-stack` rules to feed through but now the page format is properly sorted they've become redundant. Will remove in the next commit :)